### DISCUSSION TITLE
Memory view support

### DISCUSSION THREAD

**Discussion by romainthomas** [wjakob/nanobind#233] [Ideas]
Thank you for making this new library!

Would it make sense to have a `nb::memoryview` in nanobind?

I tried to workaround with `ndarray` without success so if the `memoryview` aims being supported, I will be happy to draft a PR for that.

**Comment by wjakob** [comment]
My first reaction is that we probably don't want a separate interface just specific to `memoryview`. It seems that `ndarray` could be updated to deal with it as well. This isn't on my priority list, so your help is welcomed if the feature is important to you.
